### PR TITLE
initial templating working

### PR DESCRIPTION
**Pull request by tg339**

### CODE REVIEW

**Comment by callicles** [comment]
Can you rebase on `main` you have some conflicts

File: packages/igloo-kit/src/client.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+interface Client {
```
Comment (callicles) [inline packages/igloo-kit/src/client.ts:1]: Do you still need this package then?

**Review by callicles — APPROVED** [review]
Nice work! Can't wait to give it a spin :)

File: packages/igloo-kit/src/client.ts (hunk 1 above)
Comment (tg339) [reply packages/igloo-kit/src/client.ts:1]: Not for now. This thing is just a placeholder.

**Pull request merged**